Add login-ada Slack handle to Slack article (#246)
18F/identity-handbook: _articles/slack.md

- `@login-appdev-team` Group for application development teams
- `@login_leads` login team leads
- `@login-feds` Group for Login.gov federal FTE employees

> - `@login-ada` To tag Team Ada, one of the AppDev teams that works on the IdP
> - `@login-grace` To tag Team Grace, one of the AppDev teams that works on the IdP
> - `@login-katherine` To tag Team Katherine, one of the AppDev teams that works on the IdP and the brochure site

Reviewer: one thought while we're here, what if instead of duplicating team descriptions we linked to the teams page? 
```suggestion
- `@login-ada` To tag members of [Team Ada]({% link _articles/sprint-teams.md %}#ada)
- `@login-grace` To tag members of [Team Grace]({% link _articles/sprint-teams.md %}#grace)
- `@login-katherine` To tag members of [Team Katherine]({% link _articles/sprint-teams.md %}#katherine)
```

Author: I like it! Updated in [919b232](https://github.com/18F/identity-handbook/pull/246/commits/919b232ab5aad0ab724d39c931a09d1c89e19ba2).
